pkg/timestream/macros_test.go

```diff
@@ -93,4 +93,69 @@ func TestInterpolate(t *testing.T) {
 			t.Fatalf("Result above tolerated precision %d : %d, %d", precision, numtext, expect)
 		}
 	})
+
+	t.Run("using timeFrom", func(t *testing.T) {
+		sqltxt := `$__time_from_raw_ms`
```

Comment:
Repeating string value in two different files seems a little delicate / prone to potential maintainability issues down the road.

I'm not as familiar with Go / Go conventions, but in naming a const is the naming convention camelCase vs. CONST_VAR (similar to JS / TS?). Making naming of const very clear may help some (starting on line 13 of `macros.go`) Also, are we able to export individual variables from a file? If so we could export the const string values from `macros.go` to use them with more confidence in this testing file.

Disclaimer: This may be a premature optimization / me not understanding Go :)